### DISCUSSION TITLE
Standalone FleetDM installation and FleetDM link does not appear on SOC

### DISCUSSION THREAD

**Discussion by leamee** [Security-Onion-Solutions/securityonion#7812] [General]
We have fresh distributed installation of security onion 2.3.110. While installing manager node we disabled “osquery” module and installed FleetDM as a standalone node. After the installation, we cannot see FleetDM link on the bottom left part of the SOC. We can access FleetDM from its IP but not from SOC. 
How can we fix this issue?

**Comment by defensivedepth** [comment]
Run the following on the Manager:

`sudo so-nginx-restart --force`

If that doesn't clear it up, please post a sanitized output of the following (on the Manager):

`sudo salt-call pillar.get global`

**Reply by leamee** [reply]
`sudo so-nginx-restart --force` did not worked. Here is the `sudo salt-call pillar.get global` output:

```
local:
    ----------
    airgap:
        False
    dockernet:
        172.17.0.0
    fleet_custom_hostname:
        None
    fleet_hostname:
        so-fleet
    fleet_ip:
        10.X.X.X
    fleet_manager:
        False
    fleet_node:
        True
    fleet_packages-timestamp:
        2022-04-19-09:01
    fleet_packages-version:
        3
    hnmanager:
        10.0.0.0/8,192.168.0.0/16,172.X.X.X/24,172.X.X.X/24
    ids:
        Suricata
    imagerepo:
        security-onion-solutions
    managerip:
        10.X.X.X
    mdengine:
        ZEEK
    pipeline:
        redis
    sensoronikey:
        Io4qG*************
    soversion:
        2.3.110
    url_base:
        10.X.X.X
    wazuh:
        1
```

**Comment by leamee** [comment]
I don't know if it is relevant or not but I got an error while triggering a highstate as follows:


```
[solocal@so-manager ~]$ sudo salt-call state.highstate
local:
    Data failed to compile:
----------
    Detected conflicting IDs, SLS IDs need to be globally unique.
    The conflicting ID is 'so-fleet' and is found in SLS 'base:fleet' and SLS 'base:fleet.install_package'
```

**Reply by defensivedepth** [reply]
@leamee 

Yes, this is odd. 

To confirm, you have two different installs of Security Onion: the Manager node and the Fleet Standalone node?

Have you made any edits to any salt files on the Manager?

**Reply by leamee** [reply]
Actualy I have 1 manager node, 1 sensor node, 1 search node and 1 standalone fleet node. And I did not made any edits to any salt files on the manager.

Looks like I am gonna try a reinstall.

**Reply by defensivedepth** [reply]
Can you post the output of the following (run from the Manager):

`sudo ls -l /opt/so/saltstack/local/salt/fleet/`

Also, can you post the contents of the following sls files?

`sudo cat /opt/so/saltstack/default/salt/fleet/install_package.sls`
